.pre-commit-hooks.yaml

```
@@ -15,7 +15,7 @@
   language: script
   # Supported extensions by Terraform specified in
   # https://github.com/hashicorp/terraform/blob/0c63fb2b097edcd5cb1a91322765a414206fbea2/internal/command/fmt.go#L30-L35
-  files: \.(tf|tofu|tfvars|tftest\.hcl|tfmock\.hcl)$
+  files: \.(tf|tofu|tfvars|(tftest|tofutest|tfmock|tofumock)\.hcl)$
```

Techincally, .tofumock.hcl does not exist yet, and looks like it will not be implemented soon (as they asked for community PR) - https://github.com/opentofu/opentofu/issues/1778


```suggestion
  files: \.(tf|tofu|tfvars|(tftest|tofutest|tfmock)\.hcl)$
```